app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -81,17 +81,15 @@ public function getAdditionalData(array $excludeAttr = [])
         $attributes = $product->getAttributes();
         foreach ($attributes as $attribute) {
             if ($attribute->getIsVisibleOnFront() && !in_array($attribute->getAttributeCode(), $excludeAttr)) {
-                if (is_array($value = $attribute->getFrontend()->getValue($product))) {
-                    continue;
-                }
-                if (!$product->hasData($attribute->getAttributeCode())) {
-                    $value = __('N/A');
-                } elseif ((string)$value == '') {
-                    $value = __('No');
-                } elseif ($attribute->getFrontendInput() == 'price' && is_string($value)) {
+                $value = $attribute->getFrontend()->getValue($product);
+
+                if ($value instanceof Phrase) {
+                    $value = (string)$value;
+                } if ($attribute->getFrontendInput() == 'price' && is_string($value)) {
```

Comment:
This `if` formatting looks pretty strange, should it be on next line or `elseif`? For the sake of cleanup `===` use would be nice also.

Reply:
Sure. should be `elseif`. Thanks